addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/YamahaReceiverHandlerFactory.java

```diff
@@ -8,26 +8,34 @@
  */
 package org.openhab.binding.yamahareceiver.internal;
 
-import static org.openhab.binding.yamahareceiver.YamahaReceiverBindingConstants.THING_TYPE_YAMAHAAV;
-
-import java.util.Collections;
 import java.util.Set;
 
+import org.eclipse.smarthome.config.core.Configuration;
+import org.eclipse.smarthome.core.thing.Bridge;
 import org.eclipse.smarthome.core.thing.Thing;
 import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.ThingUID;
 import org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory;
 import org.eclipse.smarthome.core.thing.binding.ThingHandler;
-import org.openhab.binding.yamahareceiver.handler.YamahaReceiverHandler;
+import org.openhab.binding.yamahareceiver.YamahaReceiverBindingConstants;
+import org.openhab.binding.yamahareceiver.handler.YamahaBridgeHandler;
+import org.openhab.binding.yamahareceiver.handler.YamahaZoneThingHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.collect.Sets;
 
 /**
  * The {@link YamahaReceiverHandlerFactory} is responsible for creating things and thing
  * handlers.
  *
- * @author David Gräff - Initial contribution
+ * @author David Graeff <david.graeff@web.de>
```

Comment:
What is the nowadays way of using the author annotations in the Oh2 addons codebase? Should I  just remove those entirely? Should I use the international version of my name (to allow people who want to contact me, to be able to type my name)? Git blame is superior in finding out what an author contributed, so should it be the name only without an email address? How does another developers contact previous authors to sort out code questions if there is now way of contacting him? (That is the reason for the author tag, isn't it?) Does the name should match the github name or the signoff name?